src/UI/Utils/Element.elm

```diff
@@ -0,0 +1,12 @@
+module UI.Utils.Element exposing (letterSpacing)
+
+import Element exposing (Attribute)
+import Html.Attributes as HtmlAttrs
+
+
+letterSpacing : Float -> Attribute msg
```

Comment:
Why create one if the [elm-ui](https://package.elm-lang.org/packages/mdgriffith/elm-ui/latest/Element-Font#letterSpacing) already has it?

Reply:
Got rid of this too
(It was reminiscent of a whole wrong thinking line)